### PR TITLE
Replaced all tags links in support panel with dfeshiny::external_link()

### LINKED ISSUE #50: Update links in cookies and support panel to use dfeshiny::external_link()

### PR DESCRIPTION
**Pull request by rmbielby**
# Brief overview of changes

We'd added the `external_link()` function previously, but not applied it in `support_panel()`. I've done a quick update to change the links in the support panel to use external_link.

## Why are these changes being made?

We want to be internally consistent in using our own advice!

## Detailed description of changes

There were several `tags$a()` instances in support links. I've updated all of these to `external_link()` and removed the warning for the ones that are mailto links to e-mail addresses.

## Connected issues
Resolves #50

### CODE REVIEW

File: R/support_panel.R [hunk 1]
```diff
@@ -173,30 +169,30 @@ support_panel <- function(
...
     shiny::tags$p(
       "If you have questions about the dashboard or data within it,
             please contact us at ",
-      shiny::tags$a(
+      dfeshiny::external_link(
         href = paste0("mailto:", team_email),
-        team_email, .noWS = c("after")
+        link_text = team_email,
+        add_warning = FALSE
       )
```
Comment (cjrace) [inline R/support_panel.R:186]: Want a full stop after this line

File: R/support_panel.R [hunk 2]
```diff
@@ -103,10 +103,9 @@ support_panel <- function(
         "This dashboard is a new service that we are developing. If you
               have any feedback or suggestions for improvements, please submit
               them using our ",
-        shiny::tags$a(
+        dfeshiny::external_link(
           href = form_url,
-          "feedback form",
-          .noWS = c("after")
+          link_text = "feedback form"
```
Comment (cjrace) [inline R/support_panel.R:108]: Want a full stop at the end of this line

File: R/support_panel.R [hunk 3]
```diff
@@ -136,35 +135,32 @@ support_panel <- function(
...
+        dfeshiny::external_link(
           href = paste0(
             "https://explore-education-statistics.service.gov.uk/find-statistics/", # nolint: [line_length_linter]
             publication_slug
           ),
-          ifelse(
+          link_text = ifelse(
             !is.null(publication_name),
             publication_name,
             "Explore Education Statistics"
```
Comment (cjrace) [inline R/support_panel.R:146]: Lower case for ees

File: R/support_panel.R [hunk 4]
```diff
@@ -136,35 +135,32 @@ support_panel <- function(
         "The parent statistical release of this dashboard, along with
               methodological information,
               is available at the following link: ",
```
Comment (cjrace) [inline R/support_panel.R:137]: I would drop some of this and change to:

`is available at [explore education statistics](link).`

**Review by cjrace — CHANGES_REQUESTED** [review]
Added a couple of comments, mostly around things you didn't touch but are quick tidy up things on this page generally

File: www/cookie-consent.js
[the remark this answers is not in the document]
Comment (rmbielby) [reply www/cookie-consent.js:1]: Turning into my calling card... Removed now and added www on to the .gitignore

File: R/support_panel.R [hunk 5]
```diff
@@ -136,35 +135,32 @@ support_panel <- function(
...
         " and ",
-        shiny::tags$a(
+        dfeshiny::external_link(
           href = paste0(
             "https://explore-education-statistics.service.gov.uk/find-statistics/", # nolint: [line_length_linter]
             publication_slug, "#explore-data-and-files"
           ),
-          "tools to access and interogate the underling data",
-          .noWS = c("after")
+          link_text = "tools to access and interogate the underling data"
```
[the remark this answers is not in the document]
Comment (rmbielby) [reply R/support_panel.R:163]: Fixed

File: R/support_panel.R (hunk 1 above)
Comment (rmbielby) [reply R/support_panel.R:186]: Done

File: R/support_panel.R (hunk 2 above)
Comment (rmbielby) [reply R/support_panel.R:108]: Done

File: R/support_panel.R (hunk 3 above)
Comment (rmbielby) [reply R/support_panel.R:146]: Done

**Review by cjrace — APPROVED** [review]

**Pull request merged**